src/test/java/com/codegym/jrugotom5/service/AdvertServiceTest.java

```diff
@@ -95,4 +97,39 @@ void testGetAdvertsByDateRange_InvalidDateRange() {
 
         assertEquals("'From' date should be after 'To' date.", exception.getMessage());
     }
+
+    @Test
+    void testGetAdvertsByCreateBy_WithAdverts_ReturnsDtoList() {
+        Long userId = 1L;
+        Advert advert1 = new Advert();
+        Advert advert2 = new Advert();
+        List<Advert> adverts = List.of(advert1, advert2);
+
+        when(advertRepository.getAdvertsByCreatedById(userId)).thenReturn(adverts);
+
+        AdvertInfoForCreatorDto dto1 = new AdvertInfoForCreatorDto();
+        AdvertInfoForCreatorDto dto2 = new AdvertInfoForCreatorDto();
+
+        when(modelMapper.map(advert1, AdvertInfoForCreatorDto.class)).thenReturn(dto1);
```

Comment:
if you have test with 30 DTOs then u will mock "mapping" all of them?
Consider to use `map(any(Advert.class), eq(AdvertInfoForCreatorDto.class))`